public/src/utils.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line
+export const formatDate = (date) => `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(2, '0')}-${String(date.getDate()).padStart(2, '0')}T${String(date.getHours()).padStart(2, '0')}:${String(date.getMinutes()).padStart(2, '0')}`;
```

Comment:
looks like you should use date-fns :)

date-fns is at least tree shakeable and pretty small in comparison to moment